lib/src/chip/chip.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component, Input, ViewEncapsulation, OnInit } from '@angular/core';
+import { anyToBoolean } from '../util';
+
+@Component({
+    selector: 'hc-chip',
+    template: `<div [ngClass]="chipType"><ng-content></ng-content></div>`,
+    styleUrls: ['./chip.component.scss'],
+    encapsulation: ViewEncapsulation.None
+})
+
+export class ChipComponent implements OnInit {
+
+    _action: boolean = false;
+    @Input() color: string = 'neutral';
```

Comment:
I see that `neutral`, `red`, `yellow`, and `green` are the color options, so we could enforce that a little more, rather than taking in any string.
```
@Input() color: 'neutral' | 'yellow' | 'green' | 'red' = 'neutral';
```
Done this way, the implementer of the component should be able to get helpful tooltips in their IDE about what is available, and errors if they use an invalid color.